src/routes/RepositoryDetails/RepositoryDetails.tsx

```diff
@@ -92,7 +96,7 @@ export default function TagsList() {
             title={<TabTitleText>Settings</TabTitleText>}
           >
             <div>Settings tab work in progress</div>
-          </Tab>
```

Comment:
instead of commenting it out, I'd say just remove the whole thing

Reply:
removed